### ISSUE TITLE
508/a11y: SVG icon for OES in footer has no alt text or accessible name

### ISSUE THREAD

**Issue by iamjolly** [gsa-oes/office-of-evaluation-sciences#335]
The OES logo is an inline SVG within the footer of the site. It has no accessible name. With the logo being inside a link, this also causes the link to have no meaningful content or description. 

Inline SVGs can be made accessible by the following methods:
* add `role="img"` attribute to the `svg` element
* add `aria-label="OES"` attribute to the `svg` element

These changes will allow the SVG to meet [WCAG 2.1 AA/508 for 1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content) and [2.4.4 Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)

Screenshot:
<img width="1022" alt="Chrome screenshot showing SVG without accessible name" src="https://user-images.githubusercontent.com/1093423/130098665-41e09658-6aef-45da-8b08-7c243459ab7e.png">

**Comment by alyssagalik** [comment]
added alt text for OES

**Comment by iamjolly** [comment]
@alyssagalik - This isn't fixed yet. Need to do both of the recommended code fixes on the `svg` element, not the `a`. I can fix this for you with a PR later today or tomorrow morning. Would you please re-open this one for now?

**Comment by alyssagalik** [comment]
sounds good thanks!
